var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -92,6 +92,8 @@ class Petsc(Package):
             description='Activates support for SuperluDist (only parallel)')
     variant('trilinos', default=False,
             description='Activates support for Trilinos (only parallel)')
+    variant('mkl-pardiso', default=False,
```

Comment:
I was not sure what the variant default should be :
maybe we want to build with mkl_pardiso if "intel-mkl" in spec ?

Reply:
I think `default=False` is a reasonable default